crates/ubrn_cli/src/ios.rs

```diff
@@ -241,3 +297,97 @@ impl IOsArgs {
         self.config.clone()
     }
 }
+
+/// A specific build target supported by the SDK.
+#[derive(Debug, Hash, PartialEq, Eq, Clone, Deserialize, Serialize)]
+#[serde(try_from = "String", into = "String")]
+pub(crate) struct Target {
+    triple: String,
+    platform: Platform,
+    description: String,
+}
+
+/// The platform for which a particular target can run on.
+#[derive(Debug, Hash, PartialEq, Eq, Clone, Serialize, Deserialize)]
+enum Platform {
+    Macos,
+    Ios,
+    IosSimulator,
+}
+
+impl Platform {
+    /// The name of the subfolder in which to place the library for the platform
+    /// once all architectures are lipo'd together.
+    fn lib_folder_name(&self) -> &str {
+        match self {
+            Platform::Macos => "macos",
+            Platform::Ios => "ios",
+            Platform::IosSimulator => "ios-simulator",
```

Comment:
Some part of me thinks that XCode might be very particular about the names of these folders, but I don't have a citation to back that up?

Reply:
The way I'm using them at the moment is to create a folder in the Rust `target` directory, as outputs of the lipo command.

The `xcodebuild -create-xcframework` then uses those files to create the `Framework.xcframework` folder.